tests/osaka/eip7692_eof_v1/eip4200_relative_jumps/test_rjump.py

```diff
@@ -609,7 +503,7 @@ def test_rjump_into_eofcreate(
         data=Container(
             sections=[
                 Section.Code(
-                    code=Op.RJUMP[1] + Op.EOFCREATE[0](0, 0, 0, 0) + Op.STOP,
+                    code=Op.PUSH0 * 4 + Op.RJUMP[1] + Op.EOFCREATE[0] + Op.STOP,
```

Comment:
Nice, I think this makes more sense.

Reply:
It was actually a bug. Previously it jumped to the first PUSH data before EOFCREATE. Also jumping over PUSHes changes stack height. But the main issue is that it is hard to tell that `(0, 0, 0, 0)` are actually 4 PUSH1 instructions. And maybe they will be changed to PUSH0 in some future update?

There are similar tests for RJUMPI and RJUMPV but without a bug. Should I got and change them too?

Reply:
Yes I think it's a good opportunity, this is simply a much better approach, and it's much more resilient too.

Reply:
Done.